### PR TITLE
ISSUE-6 Glowroot & CLI packaging for distributed & distributed-ldap

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by Arsnael** [comment]
Doc likely missing, and also the glowroot files for distributed-ldap it seems, but I can handle it from here, thanks :)

**Comment by Arsnael** [comment]
Will do ldap in an other PR

**Review by Arsnael — APPROVED** [review]

File: openpaas-james/apps/distributed-ldap/README.md [hunk 1]
```diff
@@ -1,9 +1,102 @@
...
+    environment:
+      - JAVA_TOOL_OPTIONS=-javaagent:/root/glowroot/glowroot.jar
+```
+
+### Using the CLI
+
+The [CLI](https://james.apache.org/server/manage-cli.html) can easily be used once your distributed-ldap server is running:
+
+```
+docker exec CONTAINER-ID james-cli ListDomains
```
Comment (chibenwa) [inline openpaas-james/apps/distributed-ldap/README.md:101]: s/CONTAINER-ID/james?

File: openpaas-james/apps/distributed-ldap/pom.xml [hunk 2]
```diff
@@ -100,12 +100,54 @@
...
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-glowroot-resources</id>
+                        <!-- here the phase you need -->
```
Comment (chibenwa) [inline openpaas-james/apps/distributed-ldap/pom.xml:131]: Can we remove this comment?

File: openpaas-james/apps/distributed-ldap/pom.xml [hunk 3]
```diff
@@ -100,12 +100,54 @@
...
     </dependencies>
+
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>install-jbpm</id>
```
Comment (chibenwa) [inline openpaas-james/apps/distributed-ldap/pom.xml:111]: `package-glowroot` ?

**Review by chibenwa — COMMENTED** [review]
+1

**Review by rouazana — APPROVED** [review]

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**